packages/app-check/karma.browser.conf.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// eslint-disable-next-line @typescript-eslint/no-require-imports
+const karmaBase = require('../../config/karma.base');
+
+const files = [`**/*.test.ts`];
+
+module.exports = function (config) {
+  config.set({
+    ...karmaBase,
+    browsers: ['Firefox'],
```

Comment:
No problem for this iteration, but start thinking about how to dynamically insert a browser here (env variable from the workflow maybe) for when we're able to do Safari, Edge, etc.

Reply:
Inserting the variable seems easy enough with `browsers: process.env?.BROWSERS?.split(',') ?? ['Firefox'],` 

To me the tricky part is installing the corresponding browser in the vm for github, you would need a browser to installation map.